edweiss-app/config/toast-config.tsx

```diff
@@ -0,0 +1,38 @@
+import { BaseToast, ToastConfig } from 'react-native-toast-message';
+
+const toastConfig: ToastConfig = {
```

Comment:
more functionality would be more interested, by exemple :  
`duration={3000} `

or other types of toasts (e.g., info, warning)